lib/erb_lint/cli.rb

```diff
@@ -168,18 +168,27 @@ def load_options(args)
 
     def lint_files
       if @options[:lint_all]
-        pattern = File.expand_path(DEFAULT_LINT_ALL_GLOB, Dir.pwd)
+        pattern = File.expand_path(glob, Dir.pwd)
         Dir[pattern].select { |filename| !excluded?(filename) }
       else
         @files
-          .map { |f| Dir.exist?(f) ? Dir[File.join(f, DEFAULT_LINT_ALL_GLOB)] : f }
+          .map { |f| Dir.exist?(f) ? Dir[File.join(f, glob)] : f }
           .map { |f| f.include?('*') ? Dir[f] : f }
           .flatten
           .map { |f| File.expand_path(f, Dir.pwd) }
           .select { |filename| !excluded?(filename) }
       end
     end
 
+    def glob
+      if File.exist?(config_filename)
+        config = RunnerConfig.new(file_loader.yaml(config_filename), file_loader)
+        @config = RunnerConfig.default.merge(config)
+        return @config.to_hash["glob"]
+      end
```

Comment:
At the point that `glob` is called, `load_config` has already been called (in `initialize`), so no need to load the config again.

Reply:
I had originally semi-redone the config load as `glob` is used [during the options parse](https://github.com/Shopify/erb-lint/pull/143/files#diff-a989663d2711ff35a3a5fc5aa185f838R262) which happens before `load_config`.

Another option I explored was moving the load_config call to run before opts, however this has knock on effects.

Namely, the LinterRegistry loads custom linters into @options and this is necessary for `runner_config_override` to function properly.

I have a fix I am working on that will make this more readable, please wait before merge.